packages/opencensus-web-instrumentation-perf/src/initial-load-root-span.ts

```diff
@@ -92,6 +93,7 @@ export function getInitialLoadRootSpan(
   root.annotations = getNavigationAnnotations(perfEntries);
   root.attributes[ATTRIBUTE_HTTP_URL] = navigationUrl;
   root.attributes[ATTRIBUTE_HTTP_USER_AGENT] = navigator.userAgent;
+  root.attributes[ATTRIBUTE_INITIAL_LOAD_TRACE_ID] = root.traceId;
```

Comment:
Could you add that this is included to enable trace search by attribute to find an initial load trace and its interaction traces via a single attribute query?